projects/core/src/auth/user-auth/services/oauth-lib-wrapper.service.ts

```diff
@@ -22,15 +23,35 @@ import { AuthConfigService } from './auth-config.service';
 })
 export class OAuthLibWrapperService {
   events$: Observable<OAuthEvent> = this.oAuthService.events;
+  currentBaseSite: BaseSite;
```

Comment:
I am trying to avoid introduce new flag. Maybe use `this.oAuthService.responseType==='code'` instead of `if (this.currentBaseSite.cdcSiteConfig) {` in [line166](https://github.com/SAP/spartacus/pull/19144/files/5caa4eb8c164e8b47e87b5fc0bc190d93c0d6043#diff-d359f9c3ad517cd184767bb6917d16624704a7b4430cceaaa968ffa457e97301R166).